ping-pong/README.md

```diff
@@ -0,0 +1,46 @@
+<p align="center">
+  <img src="https://www.corda.net/wp-content/uploads/2016/11/fg005_corda_b.png" alt="Corda" width="500">
+</p>
+
+# Ping-Pong CorDapp
+
+This CorDapp allows a node to ping any other node on the network that also has this CorDapp installed.
+
+It demonstrates how to use Corda for messaging and pass data using a flow without saving any states or using any contracts.
+
+# Pre-requisites:
+  
+See https://docs.corda.net/getting-set-up.html.
+
+# Usage
+
+## Running the nodes:
+
+See https://docs.corda.net/tutorial-cordapp.html#running-the-example-cordapp.
```

Comment:
Is this really still applicable now?

Reply:
Probably not, but it appears in most of the other samples. Maybe a subsequent pull request can address that.